js/core/modules_registry.js

```diff
@@ -0,0 +1,21 @@
+const registry = {};
+
+export function setModule(name, module) {
+    registry[name] = module;
+}
+export function getModule(name) {
+    return registry[name];
+}
```

Comment:
Exporting `getModule` and `componentGetter` in the same file may lead to incorrect API usage.
```suggestion
function getModule(name) {
    return registry[name];
}
```

Reply:
The `getModule` is a base functionality. We'll use it for not-component modules. And the `componentGetter ` is an additional helper.